src/App.jsx

```diff
@@ -4,25 +4,30 @@ import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
-class App extends React.Component {
+export class App extends React.Component {
   state = {
-    todos: [],
     selectedUserId: 0,
   };
 
   render() {
-    const { todos, selectedUserId } = this.state;
+    const { selectedUserId } = this.state;
 
     return (
       <div className="App">
         <div className="App__sidebar">
-          <TodoList todos={todos} />
+          <TodoList
+            selectUser={(userId) => {
+              this.setState({ selectedUserId: userId });
```

Comment:
For a better reading, you can create the function `selectUser` and just pass it here
`selectUser={selectUser}`